### PR TITLE
Add build and test release_latest workflow (generated by script)

### PR DESCRIPTION
**Pull request by dabonnie**
Add build and test release_latest workflow. Automatically generated by a script.

### CODE REVIEW

File: .github/workflows/build_and_test_release_latest.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          ros_version: 1
+          extra_cmake_args: --cmake-target tests
+        - ros_distro: dashing
+          ubuntu_distro: bionic
+          ros_version: 2
+          extra_cmake_args: ""
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.ros_distro }}-ros-base-latest
+    outputs:
+      build_result: ${{ steps.build.outcome }}
```
Comment (emersonknapp) [inline .github/workflows/build_and_test_release_latest.yml:36]: I don't think this `outputs` is doing anything, there is no step with `name: build` (or `test`), and the named outputs `build_result` and `test_result` aren't being read anywhere.

**Review by emersonknapp — APPROVED** [review]
This looks good assuming passing build

Would be good to remove no-op variable setting, but not blocking on that

File: .github/workflows/build_and_test_release_latest.yml (hunk 1 above)
Comment (dabonnie) [reply .github/workflows/build_and_test_release_latest.yml:36]: This is leftover from the  log to CloudWatch action (currently removed as we haven't added the secrets to this repo yet).

**Review by jikawa-az — APPROVED** [review]

File: .github/workflows/build_and_test_release_latest.yml (hunk 1 above)
Comment (emersonknapp) [reply .github/workflows/build_and_test_release_latest.yml:36]: does that portion actually use these variables, or is it dangling from an older rendition of the reporting code? E.g. https://github.com/aws-robotics/rosbag-uploader-ros1/blob/master/.github/workflows/main.yml does not use `build_result` anywhere

**Pull request merged**